src/OidcClient.php

```diff
@@ -266,15 +290,12 @@ private function getConfigurationValue(string $key): mixed
    *
    * @throws OidcException
    */
-  private function requestTokens(string $code): OidcTokens
+  private function requestTokens(array $params): OidcTokens
```

Comment:
I'm in doubt about this change. I think I prefer the method to be as strictly defined as possible, so something like:

```php
private function requestTokens(string $grantType, string $code = null, string $refreshToken = null, string $redirectUrl = null): OidcTokens
```

That way, we can utilise named parameters in the call, and only add the param when it has been set.

What do you think?